src/planet.ts

```diff
@@ -98,6 +101,7 @@ class Planet {
   update(camera: PlanetCamera) {
     // Make the planet mesh and all of its child meshes turn to look at the new camera position.
     this.mesh.lookAt(camera.position);
+    this.mesh.visible = false;
```

Comment:
The idea is that we have two different meshes: the one with the Voronoi cells that we use for calculating the high-level tectonic plate details, which we don't usually show, and the planet mesh with the terrain drawn on it that we show to the user. While we're working on the plate generation stuff, though, we should make the planet mesh invisible and show the underlying data instead.